sci-biology/afni/afni-0_pre20130927.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit multilib pax-utils toolchain-funcs
+
+DESCRIPTION="An open-source environment for processing and displaying functional MRI data"
+HOMEPAGE="http://afni.nimh.nih.gov/"
+SRC_URI="http://dev.gentoo.org/~jlec/distfiles/${P}.tgz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+RDEPEND="dev-libs/expat
+	media-libs/libjpeg-turbo
```

Comment:
AFNI ships two files which we package as part of libjpeg-turbo:

* `/usr/bin/cjpeg`
* `/usr/bin/djpeg`

 I am assuming afni will expect to find the things they ship it with. if all other things which `virtual/jpeg` could point to ship these files we can use that instead.